scrapy/downloadermiddlewares/decompression.py

```diff
@@ -4,18 +4,14 @@
 
 import bz2
 import gzip
+from io import BytesIO
```

Comment:
Nitpick: could you group the `^import ...` and `^from ...` lines together? There are a couple other occurrences of this in PR.

Reply:
All `^import` together and then all `^from` together? I've noticed only recently that @Gallaecio is promoting this style, I didn't use it before.

Reply:
I must say I didn’t use it either until recently. I think I found about it in some pull request, and https://stackoverflow.com/a/20763446/939364 seems to confirm that this is the convention.